src/errors/BaseError.ts

```diff
@@ -0,0 +1,11 @@
+class BaseError extends Error {
+  status: number
+  message: string
```

Comment:
You don't need to declare a message here, because it's already exist in Error type.

Reply:
24dcb7919497f64c2c67f0a5f4a3865fb3c8f60f